components/ActivityCard.js

```diff
@@ -1,66 +1,53 @@
-import Link from "next/link";
+import { uid } from "uid";
 import Image from "next/image";
 import styled from "styled-components";
 
 export default function ActivityCard({ title, categories, imageUrl }) {
   return (
     <StyledCardSection>
-      <StyledImageSection>
+      <StyledImageDiv>
         <Image
           src={imageUrl}
           alt={title}
           style={{ objectFit: "cover" }}
           sizes="33vw"
           fill
         />
-      </StyledImageSection>
-      <StyledDetailsSection>
-        <StyledList>
-          {categories.map((category, index) => (
-            <StyledListItem key={index}>{category}</StyledListItem>
-          ))}
-        </StyledList>
-        <StyledLink href="#" title={title}>
-          {title}
-        </StyledLink>
-      </StyledDetailsSection>
+      </StyledImageDiv>
+      <StyledList>
+        {categories.map((category) => (
+          <StyledListItem key={uid()}>{category}</StyledListItem>
```

Comment:
this solution does not work either, every time your code renders the key will change. Please use an information that:
1. can be found on the element that is currently passed into the callback function (in this case "category")
2. that is unique